CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing 
+
+Contributions are **welcome** and will be fully **credited**. 
+
+We accept contributions via Pull Requests on [Github](https://github.com/:vendor/:package_name). 
+
+## Pull Requests 
+
+- **[PSR-2 Coding Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)** - Check the code style before committing changes. 
+- **Add tests!** - Your patch won't be accepted if it doesn't have tests. 
+- **Document any change in behaviour** - Make sure the `README.md` and any other relevant documentation are kept up-to-date. 
+- **Create feature branches** - Don't ask us to pull from your master branch. 
+- **One pull request per feature** - If you want to do more than one thing, send multiple pull requests. 
+- **Send coherent history** - Make sure each individual commit in your pull request is meaningful. If you had to make multiple intermediate commits while developing, please [squash them](http://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages) before submitting. 
+
+
+##Happy coding!
```

Comment:
Missing space after `##`

Reply:
:+1: Improved.

Reply:
Did you push fix? I can't see it.

Reply:
There are no specifications for styling md file but I think empty line on the end of file will be helpful to view changes in PR. I added the new line but github not see difference, if you click edit file and   you could see raw file with changes

Reply:
I have no permission to edit file so I cannot see this :(
Anyway I think we are talking about two different things. You mentioned space at the end and I meant space between `##` and `Happy coding!` in the last line :)

Reply:
Ahh I see - now should be correct. Thanks!